cpp/lib/include/LobidUtil.h

```diff
@@ -33,6 +33,8 @@ namespace LobidUtil {
  *         - http://lobid.org/resources/api
  */
 std::string GetAuthorGNDNumber(const std::string &author, const std::string &additional_query_params="");
+std::string GetAuthorLastFirstGNDNumber(const std::string &author_surname, const std::string &author_firstname,
```

Comment:
I think a better name would be GetAuthorLastFirstAndGNDNumber

Reply:
The Function only returns a GNDNumber but unlike the original one uses separate first und last name in the argument list.

Reply:
Ok, then how about GetGNDNumberFromAuthorName?

Reply:
This would break the analogy to `GetAuthorGNDNumber` above or this would have to be renamed as well and the code base adjusted.

Reply:
I am for keeping that name because I don't think it is confusing at all.  The new name strikes me as confusing though.

Reply:
How about returning to my original approach of overloading `GetAuthorGNDNumber` and removing the default parameter in the third argument to make the call unique?

Reply:
How about just overloading GetAuthorGNDNumber?

Reply:
That's the suggestion and was the original approach. However, one would have to do away with the default parameter `additional_query_params=""` because otherwise the compiler cannot properly disambiguate the first two `std::string` arguments.